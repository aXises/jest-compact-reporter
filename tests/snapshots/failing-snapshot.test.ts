it("Should not match snapshot", () => {
    expect("invalid").toMatchSnapshot();
});
