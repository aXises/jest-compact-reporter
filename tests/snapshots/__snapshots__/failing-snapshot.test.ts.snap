// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should not match snapshot 1`] = `"mysnapshot"`;
